main/states/busy_state.py

```diff
@@ -131,10 +135,15 @@ def on_exit(self):
             print("Only available for devices with RPI.GPIo ports")
 
     def __speak(self, text):
+        base_folder = os.path.dirname(os.path.abspath(__file__))
+        tts_folder = os.path.join(base_folder, '../speech/tts_wrapper.sh')
```

Comment:
Is this _../speech/tts_wrapper.sh_ a file or folder?

Reply:
@hongquan , it's a file